src/ir/assertions.cpp

```
@@ -47,6 +46,7 @@ class AssertExtractor {
 
     vector<Assertion> operator()(const LoadMapFd&) const { return {}; }
     vector<Assertion> operator()(const LoadMapAddress&) const { return {}; }
+    vector<Assertion> operator()(const LoadPseudo&) const { return {}; }
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)